docs/reference/supported_apis.md

```diff
@@ -9,18 +9,18 @@
 
 |     **CUDA**     | **HIP**                                                           |                            **ROC**                        |                            **HIP & ROC**                                |
 |:-----------------|:------------------------------------------------------------------|:----------------------------------------------------------|:------------------------------------------------------------------------|
-| CUDA Runtime API | [HIP API](tables/CUDA_Runtime_API_functions_supported_by_HIP.md)  |                                                           |                                                                         |
-| CUDA Driver API  | [HIP API](tables/CUDA_Driver_API_functions_supported_by_HIP.md)   |                                                           |                                                                         |
-| CUComplex API    | [HIP API](tables/cuComplex_API_supported_by_HIP.md)               |                                                           |                                                                         |
-| CUDA Device API  | [HIP Device API](tables/CUDA_Device_API_supported_by_HIP.md)      |                                                           |                                                                         |
-| CUDA RTC API     | [HIP RTC API](tables/CUDA_RTC_API_supported_by_HIP.md)            |                                                           |                                                                         |
+| CUDA Runtime API | [HIP API](tables/CUDA_Runtime_API_functions_supported_by_HIP.md)  |    |    |
+| CUDA Driver API  | [HIP API](tables/CUDA_Driver_API_functions_supported_by_HIP.md)   |    |    |
+| CUComplex API    | [HIP API](tables/cuComplex_API_supported_by_HIP.md)               |    |    |
+| CUDA Device API  | [HIP Device API](tables/CUDA_Device_API_supported_by_HIP.md)      |    |    |
+| CUDA RTC API     | [HIP RTC API](tables/CUDA_RTC_API_supported_by_HIP.md)            |    |    |
 | CUBLAS API       | [HIP BLAS API](tables/CUBLAS_API_supported_by_HIP.md)             | [ROC BLAS API](tables/CUBLAS_API_supported_by_ROC.md)     | [HIP + ROC BLAS API](tables/CUBLAS_API_supported_by_HIP_and_ROC.md)     |
 | CUSPARSE API     | [HIP SPARSE API](tables/CUSPARSE_API_supported_by_HIP.md)         | [ROC SPARSE API](tables/CUSPARSE_API_supported_by_ROC.md) | [HIP + ROC SPARSE API](tables/CUSPARSE_API_supported_by_HIP_and_ROC.md) |
-| CUSOLVER API     | [HIP SOLVER API](tables/CUSOLVER_API_supported_by_HIP.md)         |                                                           |                                                                         |

```

Comment:
1. We do not support cuSOLVER to rocSOLVER hipification officially. Please remove ROC-related cells.
2. Please restore the formatting here unless it is not obligatorily needed.

Reply:
It is a manually edited table, not generated, so for manual support, I'd appreciate keeping the old formatting. Thank you!